charmbits/service/server.go

```diff
@@ -49,29 +45,51 @@ type Context struct {
 	socketPath string
 }
 
+type rpcCommand struct {
+	listener net.Listener
+}
+
+func (c *rpcCommand) Kill() {
+	c.listener.Close()
+}
+
+func (c *rpcCommand) Wait() error {
+	return nil
```

Comment:
I'm just wondering why this doesn't need to block. :) I think it's probably a bug that it doesn't.